### PR TITLE
Replace log_error() by log_fatal_error()

### PR DESCRIPTION
**Pull request by emmawu**
Replaced all existing usage or log_error() with log_fatal_error(). Test will only be aborted if a fatal error is reported.

- log_fatal_error() - It will print the error message, report fail on current test task and the whole test set will be aborted.
- log_error() - It will print the error message, report fail on current test task. But it won't abort the whole test set.
- log_warn() - It will print the warn message only. No test will be terminated.

Signed-off-by: xiawu <xiawu@redhat.com>

### CODE REVIEW

File: lib/log.sh [hunk 1]
```diff
@@ -92,14 +92,23 @@ log_warn()
     log "WARN" "$@"
 }
 
-
-# @usage: log_error <mesg>
+# @usage: log_fatal_error <mesg>
```
Comment (sztsian) [inline lib/log.sh]: why line 95 and line 105 all commented as log_fatal_error...

Comment (emmawu) [inline lib/log.sh]: updated.

Comment (sztsian) [inline lib/log.sh]: I did not see any changes.

Comment (emmawu) [inline lib/log.sh]: pls check it again

**Pull request merged**